### ISSUE TITLE
Corsair H150i Pro XT Not Detected

### ISSUE THREAD

**Issue by mermelmadness** [iJacks1980/FanControl.CommanderPRO#5]
Similar to #3 my AIO is not detected.  Note that I am not routing the AIO through a Commander Pro, but rather directly to my motherboard's USB header.  I didn't know if I should try similar steps that are in #3 since the hardware and configuration are different.  I have iCue installed but typically do not run it, not even in the background.

**Comment by irlrobins** [comment]
I believe your talking about the H150i RGB PRO XT cooler? Which doesn't have any commander. Instead I think the three fans connect to a splitter cable that in turn connects to the CPU Fan header? In which case the CPU Fan header would control the fans. I think the USB header connection just powers the pump.

**Comment by mermelmadness** [comment]
Not exactly.  The fans do connect to a 3-to-1 splitter cable, but the cable connects directly to the pump, not a fan header.

**Comment by irlrobins** [comment]
Ok, manual shows a connection to a fan header, so I assumed that was a 3-to-1 into motherboard.

**Comment by mermelmadness** [comment]
Not on mine.  Might be an older manual.  Even without physically looking I can tell you that Fans 1-4, and 6-7 don't do anything.  Fan #1 reads something, but when I manually control it and set it to 0 the fans don't stop.

**Comment by iJacks1980** [comment]
Sadly the problem with this device is that it is not an iCUE Commander, it has it's own direct protocol.

I will see if I can find said protocol, but can't promise anything at the moment.

**Comment by mermelmadness** [comment]
No problem.  Thankfully, even with iCue not running the fans remembers the curve settings I last set in iCue.
